### PR TITLE
Sort lang entries in config files

### PR DESCRIPTION
**Pull request by jihoon-seo**
This PR is to sort language entries in ascending alphabetical order.
This will make it easy to
- find existing language entry
- add new language entry in proper location

### CODE REVIEW

**Comment by seokho-son** [comment]
Please hold until #482 merged.

**Comment by jihoon-seo** [comment]
- Force-pushed to resolve the conflict with #482.
- Not relevant with sorting, but GitHub CODEOWNERS checker says that this CODEOWNERS file contains errors.

> Unknown owner on line 28: make sure [**Giulia-dipietro**](https://github.com/Giulia-dipietro) exists and has write access to the repository
> /content/it/ [**Giulia-dipietro**](https://github.com/Giulia-dipietro) meryem-ldn annalisag-spark sistella
> 
> Unknown owner on line 31: make sure [**AShabana**](https://github.com/AShabana) exists and has write access to the repository
> …/ar/ TarekMSayed same7ammar [**AShabana**](https://github.com/AShabana) hacktron95

**Review by iamNoah1 — APPROVED** [review]
super hard to review because of the view in github, but the sorting itself looks good.

File: CODEOWNERS
Comment (seokho-son) [reply CODEOWNERS:31]: @iamNoah1 

It is because the invitations for @AShabana and @Giulia-dipietro are pending.
If they don't accept it, we cannot give write permissions to them. I think we need to check whether they are still willing to  be an approver.

Comment (seokho-son) [reply CODEOWNERS:31]: Thanks ! :)

**Review by iamNoah1 — COMMENTED** [review]
@jihoon-seo do we still need this?

**Comment by jihoon-seo** [comment]
@iamNoah1 There seems no strong requirement to keep the entries alphabetically ordered.
Let me close this PR. 😊

**Pull request closed without merging**
